src/solvers/dg_p4est/containers.jl

```diff
@@ -162,6 +162,13 @@ function init_interfaces(mesh::P4estMesh, equations, basis, elements)
 end
 
 
+function init_interfaces!(interfaces, mesh::P4estMesh)
+  init_surfaces!(interfaces, nothing, nothing, mesh)
+
+  return interfaces
+end
+
+
 mutable struct BoundaryContainerP4est{NDIMS, uEltype<:Real, NDIMSP1} <: AbstractContainer
```

Comment:
Why `BoundaryContainerP4est` and not `P4estBoundaryContainer` (compare `P4estMesh` vs `MeshP4est`)?